packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2369,7 +2370,16 @@ const schema: SchemaType<DbUser> = {
 
   'moderatorActions.$': {
     type: 'Object'
-  }
+  },
+  subforumPreferredLayout: {
+    type: String,
+    allowedValues: subforumLayouts,
+    hidden: true, // only editable from SubforumActionsButton
+    optional: true,
+    canRead: [userOwns, 'admins'],
+    canCreate: ['members', 'admins'],
+    canUpdate: [userOwns, 'admins'],
+  },
```

Comment:
Initially this was on UserTagRels, and then I moved it to the user because:
 - Loading either of the layouts required the userTagRel to be loaded, which was slower, and also hard to do in a way that didn't flicker one layout before showing the other. This was the main reason
 - Also I would guess that people would usually want the layout to persist across all subforums? I'm not as sure about this though

Reply:
I would guess # 2 is false sadly. Not sure though. Hmm. Probably don't care enough to block.